Lib/typing.py

```diff
@@ -3783,6 +3784,44 @@ def get_protocol_members(tp: type, /) -> frozenset[str]:
     return frozenset(tp.__protocol_attrs__)
 
 
+_P = ParamSpec("_P")
+
+
+def copy_kwargs(
```

Comment:
If this is added to CPython, it should use PEP 695 syntax:
```suggestion
def copy_kwargs[P, T](
```

Reply:
Changed it to PEP 695 syntax and renamed the params in the same step.